StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1130CodeFixProvider.cs

```diff
@@ -216,19 +216,22 @@ private static List<ParameterSyntax> GenerateUniqueParameterNames(SemanticModel
         {
             var parameters = new List<ParameterSyntax>();
 
-            foreach (var argumentName in argumentNames)
+            if (!argumentNames.IsDefault)
```

Comment:
:question: would it be cleaner to change the initial value to `Empty` here?

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/53df141371b0e4767233dc4accd3a92b1199f144/StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1130CodeFixProvider.cs#L79

Reply:
That is better yes. My initial thoughts were that a difference between a non-existing parameter list and an empty parameter list was necessary, but that is not the case.